Mindo-Joseph/CalculatorProject
# Feature/milestone 1 project setup

#### What does this PR do?
- It creates a React App using create-react-app and deploys it to Heroku.

#### Description of Task to be completed?
You will build a calculator for Math-magicians Inc using create-react-app, a tool built by developers at Facebook to help you build React applications, saving you from time-consuming setup and configuration.

#### How should this be manually tested?
Visit the following [link](https://immense-refuge-69310.herokuapp.com/) to confirm Heroku deployment.
